Modules/@babylonjs/react-native-windows/package.json

```diff
@@ -25,11 +25,11 @@
   "readmeFilename": "README.md",
   "peerDependencies": {
     "@babylonjs/core": ">=5.53.1",
-    "@babylonjs/react-native": "version",
-    "react": ">=17.0.1",
-    "react-native": ">=0.64.0",
-    "react-native-permissions": "^3.0.0",
-    "react-native-windows": ">=0.64.1"
+    "@babylonjs/react-native": "*",
+    "react": "*",
+    "react-native": "*",
+    "react-native-permissions": "*",
+    "react-native-windows": "*"
```

Comment:
Even if I got RNTA to compile, `react-native-permissions` was erroring on startup, removing exact version of peer dependency fixed the issue. Probably there is other way to this let me know if I should change it